### PR TITLE
Migrate storage to new architecture

### PR DESCRIPTION
**Pull request by pchaussalet**

### CODE REVIEW

File: core/service/section/abstract-section-service.js [hunk 1]
```diff
@@ -37,6 +39,11 @@ exports.AbstractSectionService = Montage.specialize({
         }
     },
     
+    loadExtraEntries: {
+        value: function() {
```
Comment (thibaultzanini) [inline core/service/section/abstract-section-service.js:43]: Can be replaced by `Function.noop` (optimisation)

File: ui/sections/storage/inspectors/volume.reel/volume.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+                if (datasets[i].id === volume.id) {
+                    return datasets[i];
+                }
+            }
+        }
+    },
+
+    handleExportAction: {
+        value: function () {
+            this.object.services.export(this.object.id);
```
Comment (thibaultzanini) [inline ui/sections/storage/inspectors/volume.reel/volume.js]: not in a service?

File: ui/sections/storage/inspectors/volume.reel/volume.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+        }
+    },
+
+    handleScrubAction: {
+        value: function () {
+            this.object.services.scrub(this.object.id);
```
Comment (thibaultzanini) [inline ui/sections/storage/inspectors/volume.reel/volume.js]: Same question here.

**Review by thibaultzanini — COMMENTED** [review]
LGTM, just some little comments.

File: core/service/section/abstract-section-service.js (hunk 1 above)
Comment (pchaussalet) [reply core/service/section/abstract-section-service.js:43]: not until `Function.noop` is part of standard JS `Function` object... ;)

**Pull request merged**